src/pages/index.js

```diff
@@ -79,7 +79,6 @@ const IndexPage = ({
       </LatestPosts>
       <GreyBackground>
         <Jobs />
-        <Padding bottom={{ smallPhone: 1.5, smallTablet: 0 }} />
```

Comment:
This is removed but we're now missing the spacing in the view. 

`const StyledJobs = styled(Jobs)`{/*...padding...*/}` `

Reply:
We decided that this Padding is not necessary.

Reply:
As Semra pointed out, we're not. My bad